Boost/build.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
```

Comment:
This file and the patch file are exactly the same as the ones we have in client.
I was expecting to find some different flag solving the "bitcode" issue or similar.

How has been the "bitcode" issue addressed then? As I don't see anything different or new in this PR...

Reply:
the solution is in the [PR in the client repo](https://github.com/NetsoftHoldings/hubstaff-client/pull/2027). It consists of 2 elements:
1. Recent Xcode no longer adds bitcode to libs by default
2. The `bitcode_strip` tool was triggered by a separate [install_libraries.sh](https://github.com/NetsoftHoldings/hubstaff-client/pull/2027/files#diff-9cc438890f161d864ff48a63423951f71fe674f336c5c656e233e6b178256130) script, erroneously messing with the libs files.

Reply:
but then why are all the shell scripts needed here?

Then I would remove the scripts and patches from here, leaving only the updated built artifacts.
The rest (how the build is done) is being modified by Michal now and will have quite a large impact, so I would for now keep it contained in the client's repo until that is finished.

Current work for that is to separate all the build scripts we currently use (all libraries, all OSes), and all the prebuilt libs we use, and move them into a single "hubstaff-prebuilt-third-party" repo, where all scripts will reside.
All of them will be removed from client, and will only reside there.
All the other repos with libraries will be removed.

That way, it is easy to update libraries, versions and so, since all of them reside in a single place.

So for now I would leave this repo just with the updated artifacts, with nothing else, furthermore when it is just copies of files we intend to move/replace.

Reply:
they have been removed.